x-pack/plugins/cases/public/components/case_view/components/case_view_activity.tsx

```diff
@@ -229,22 +235,20 @@ export const CaseViewActivity = ({
             selectedSeverity={caseData.severity}
             onSeverityChange={onUpdateSeverity}
           />
-          {caseUsers?.reporter ? (
-            <UserList
-              dataTestSubj="case-view-user-list-reporter"
-              theCase={caseData}
-              headline={i18n.REPORTER}
-              users={[caseUsers.reporter]}
-              userProfiles={userProfiles}
-            />
-          ) : null}
+          <UserList
+            dataTestSubj="case-view-user-list-reporter"
+            theCase={caseData}
+            headline={i18n.REPORTER}
+            users={reporterAsArray}
+            userProfiles={userProfiles}
+          />
           {caseUsers?.participants ? (
             <UserList
               dataTestSubj="case-view-user-list-participants"
               theCase={caseData}
               headline={i18n.PARTICIPANTS}
               loading={isLoadingUserActions}
-              users={caseUsers.participants}
+              users={[...caseUsers.participants, ...caseUsers.assignees]}
```

Comment:
Does the ternary operator on line 245 need to be updated to reflect the `assignees` too?

How about we combine the array above this component and just check to see if it's not empty? Although is it even possible to have assignees but not a participant 🤔 that'd mean we don't have the reporter's information I suppose. Either probably best to check just in case the backend has a bug.